scripts/release.rb

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+require 'git'
+require 'octokit'
+require 'filemagic'
+
+REPO = 'cozy/cozy-stack'
+filemagic = FileMagic.new FileMagic::MAGIC_MIME
+root_dir = File.dirname File.dirname File.expand_path __FILE__
+
+git = Git.open root_dir
+tag = git.describe '--tags'
+
+token = ENV.fetch 'GITHUB_RELEASE_TOKEN'
+github = Octokit::Client.new access_token: token
+user = github.user
+user.login
+
+# releases = github.releases REPO
+# releases.each { |r| github.delete_release r.url }
```

Comment:
Is this commented code for when an upload failed and you want to recreate the release?

Reply:
For testing purpose.
Very tedious to have to manually destroy release to be able to rerun the script without failure…
Can be removed